components/cli/pkg/commands/login.go

```diff
@@ -101,3 +125,158 @@ func RunLogin(registryURL string, username string, password string) {
 	spinner.Stop(true)
 	util.PrintSuccessMessage(fmt.Sprintf("Successfully logged into Registry: %s", util.Bold(registryURL)))
 }
+
+// This function is to request credentials from the user
```

Comment:
The normal practice in GoDocs is to have it in the format `requestCredentials requests the credentials from the user`. Note how the Doc starts with the method name